src/services/user/hooks/index.js

```diff
@@ -1,8 +1,25 @@
 const { authenticate } = require('@feathersjs/authentication');
-const errors = require('@feathersjs/errors');
+const { BadRequest, Forbidden } = require('@feathersjs/errors');
+const { iff, isProvider } = require('feathers-hooks-common');
 const logger = require('../../../logger');
-const globalHooks = require('../../../hooks');
-const { sortRoles } = require('../../role/utils/rolesHelper');
+const {
+	arrayIncludes,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['arrayIncludes' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5220864)